isaric/parsers/recover-eu_parser.toml

```diff
@@ -0,0 +1,325 @@
+[subject.has_dementia] # checkbox field
+  field = "CNEURO_DEMENT"
+  description = "Chronic Neurologic Disease -  Dementia / Cognitive dysfunction"
+  ref = "Y/N"
+
+[subject.has_rheumatologic_disorder] # checkbox field
+  field = "COTH_RHEUM"
+  description = "Chronic Other Disease -  Rheumatologic/autoimmune disease"
+  ref = "Y/N"
+
+[subject.has_hiv]
+  field = "CIMMUN_HIV"
+  description = "D. Chronic Immunosuppression: HIV"
+  ref = "Y/N"
+
+[subject.has_hypertension]
+  field = "CARD_HYPER"
+  description = "B. Chronic Cardiovasculair Disease : Hypertension"
+  ref = "Y/N"
+
+[subject.has_malignant_neoplasm]
+  combinedType = "any"
+  fields = [
+    { field = "CIMMUN_MALIGN_METAS", description = "Chronic Immunosuppression - Any malignancy within 5 years - metastatic?", ref = "Y/N" },
+    { field = "CIMMUN_MALIGN", description = "Chronic Immunosuppression - Any malignancy within 5 years", ref = "Y/N" }
+  ]
+  
+[subject.has_smoking]
+  field = "smoke"
+  description = "Smoking:	"
+
+  [subject.has_smoking.values]
+    3 = "yes" # 3="Current smoker (>=1 cigarette per day)"
+    2 = "former" # 2="Stopped smoking but used to smoke regularly"
+    1 = "never" # 1="Never smiked regulary"
+
+[subject.has_asthma] # checkbox field
+  field = "CPD_AST"
+  description = "A. Chronic Pulmonary Disease: Asthma"
+  ref = "Y/N"
+
+[subject.has_chronic_cardiac_disease]
+  field = "chr_card"
+  description = "Chronic Cardiovascular Disease"
+  ref = "Y/N"
+
+[subject.has_chronic_kidney_disease]
+  field = "COTH_RENAL"
+  description = "Chronic Other Disease -  Chronic Renal disease"
+  ref = "Y/N"
+
+[subject.has_diabetes] # checkbox fields
+  combinedType = "any"
+  fields = [
+    { field = "CMETAB_DB1", description = "C. Chronic Metabolic/Endocrine Disease : Diabetes type 1", ref = "Y/N" },
+    { field = "CMETAB_DB2A", description = "C. Chronic Metabolic/Endocrine Disease : Diabetes type 2, medicated (only oral medication)", ref = "Y/N" },
+    { field = "CMETAB_DB2B", description = "C. Chronic Metabolic/Endocrine Disease : Diabetes type 2, medicated (insulin dependent)", ref = "Y/N" }
+  ]
+
+[subject.has_liver_disease] # checkbox field
+  field = "COTH_LIVER"
+  description = "F. Chronic Other Disease : Chronic Liver disease"
+  ref = "Y/N"
+
+[subject.has_solid_organ_transplant] # checkbox field
+  field = "CIMMUN_ORG_TRANS"
+  description = "Chronic Immunosuppression - Solid organ transplant"
+  ref = "Y/N"
+
+[subject.has_hiv_art]
+  field = "CIMMUN_HIV_ANTI_THER"
+  description = "Chronic Immunosuppression - HIV - Currently on antiretroviral therapy?"
+  ref = "Y/N"
+
+[subject.has_immunosuppression]
+  field = "chr_immuno"
+  description = "Chronic Immunosuppression (due to disease)"
+  ref = "Y/N"
+
+[subject.has_asplenia]
+  field = "CIMMUN_ASPL"
+  ref = "Y/N"
+
+[subject.has_comorbidity_other] # type - array
+  combinedType = "set"
+    excludeWhen = "none"
+    fields = [
+      { field = "chr_other" },
+      { field = "COTH_OTHER" },
```

Comment:
These two have numerical (1/2 and 0/1) contents - maybe they're housekeeping variables for the later options? regardless, they shouldn't be in here. COTH_OTHER particularly is just a yes/no flag for COTH_OTHER_SPEC I think